docs/src/changelog.md

```diff
@@ -1 +1,32 @@
-../../CHANGELOG.md
\ No newline at end of file
+# Changelog
```

Comment:
Is this a soft link? Please see the `ln -s` notes in mkdocs.yaml. If that process isn't working on windows, let me know and I'll push a commit to your fork to fix

Reply:
Yes, it is. It's rendering as expected.

Reply:
The current setup appears to be a copy of the file
![Screen Shot 2022-10-31 at 14 07 19](https://user-images.githubusercontent.com/9404707/199089805-76db5ab6-7ed7-4526-85d8-916805563567.png)
rather than a soft link:
![image](https://user-images.githubusercontent.com/9404707/199089907-542af33e-9526-43cd-8e37-294e75871b44.png)

Reply:
hmm.. tried again, but it's not appearing as such.

Reply:
@JaerongA I've PR'd to your fork [here](https://github.com/JaerongA/element-lab/pull/1)

Reply:
Thanks, that's been merged.